### ISSUE TITLE
ARIA input fields do not have accessible names

### ISSUE THREAD

**Issue by datnd99** [uiwjs/react-codemirror#656]
**Describe the issue**

While running a Lighthouse audit I saw the following accessibility warning relating to the CodeMirror editor:

> ARIA input fields do not have accessible names
> 
> When an input field doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more about input field labels](https://dequeuniversity.com/rules/axe/4.6/aria-input-field-name).

You can easily reproduce it with the React CodeMirror website: https://uiwjs.github.io/react-codemirror/

Is there an option to set `aria-labels` similar to this comment?
Refer: https://github.com/codemirror/dev/issues/1127#issuecomment-1503175849

**Comment by jaywcjlove** [comment]
If you add it, how do you think about internationalization? @datnd99

**Comment by datnd99** [comment]
> If you add it, how do you think about internationalization? @datnd99

Unfortunately, In my case, I just need add **aria-label** to by pass Lighthouse test, do not require implement internationalization
